assignments/resume-bootstrap/resume.html

```diff
@@ -0,0 +1,229 @@
+                            </li>
+                            <li>
+                                <p>Django</p>
+                                <progress value="70" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>MYSQL/SQL</p>
+                                <progress value="70" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>.Net/C#</p>
+                                <progress value="60" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>Bootstrap</p>
+                                <progress value="50" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>PHP</p>
+                                <progress value="50" max="100"></progress>
+                            </li>
+
+                        </ul>
+                        <div class="maindot"></div>
+                        <h3>Personla Skills</h3>
+                        <ul class="uls">
+                            <li>
+                                <p>Communication Skills</p>
+                                <progress value="90" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>Presentation Skills</p>
+                                <progress value="90" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>Team working</p>
+                                <progress value="90" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>Planning</p>
+                                <progress value="80" max="100"></progress>
+                            </li>
+                            <li>
+                                <p>Time management</p>
+                                <progress value="90" max="100"></progress>
+                            </li>
+                        </ul>
+                    </article>
+                    <article>
+                        <h2 id="h2l">Languages</h2>
+                        <h4 class="h4L">English</h4>
+                        <h4 class="h4L">Arabic</h4>
+                        <br>
+                        <p id="p1">Excellent</p>
+                        <p id="p2">Native </p>
+                    </article>
+                    <article>
+                        <h2 class="h2l">Interests</h2>
+                        <p>Reading and teaching.</p>
+                    </article>
+                </div>
+            </main>
+        </div>
+    </div>
+    <script src="jquery.min.js"></script>
+    <!--the bootstrap depends on jquery , so the jquery hava to come above of bootstrap -->
```

Comment:
Good practice 😄 Get used to document these important but subtle things like you did here in order to help other developers and your future self.